src/main/java/friday/model/alias/AliasMap.java

```diff
@@ -0,0 +1,112 @@
+package friday.model.alias;
+
+import static friday.commons.util.CollectionUtil.requireAllNonNull;
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import friday.model.alias.exceptions.AliasNotFoundException;
+import friday.model.alias.exceptions.DuplicateAliasException;
+
+/**
+ * A map of alias to reserved keywords.
+ */
+public class AliasMap {
+
+    private final HashMap<String, String> internalMap = new HashMap<>();
```

Comment:
Can consider doing `private final Map<String, String> internalMap = new HashMap<>();`, because declaring as `Map` allows you to change the object to a different kind of `Map` besides `HashMap` without breaking the code.

Source: https://stackoverflow.com/questions/20793609/difference-between-using-map-and-hashmap-as-declared-type

Reply:
Thanks for the comment, I will do so!